app/ide-desktop/lib/client/watch.ts

```diff
@@ -72,48 +74,7 @@ const ALL_BUNDLES_READY = new Promise<Watches>((resolve, reject) => {
         console.log('Result of client bundling: ', client)
         void clientBuilder.watch()
 
-        console.log('Bundling dashboard.')
-        const dashboardOpts = dashboardBundler.bundleOptions()
-        dashboardOpts.plugins.push({
-            name: 'enso-on-rebuild',
-            setup: build => {
-                build.onEnd(() => {
-                    console.log('Dashboard bundle updated.')
-                })
-            },
-        })
-        dashboardOpts.outdir = path.resolve(IDE_DIR_PATH, 'assets')
-        const dashboardBuilder = await esbuild.context(dashboardOpts)
-        const dashboard = await dashboardBuilder.rebuild()
-        console.log('Result of dashboard bundling: ', dashboard)
-        // We do not need to serve the dashboard as it outputs to the same directory.
-        // It will not rebuild on request, but it is not intended to rebuild on request anyway.
-        // This MUST be called before `builder.watch()` as `tailwind.css` must be generated
-        // before the copy plugin runs.
-        void dashboardBuilder.watch()
-
-        console.log('Bundling content.')
-        const contentOpts = contentBundler.bundlerOptionsFromEnv({
-            supportsLocalBackend: true,
-            supportsDeepLinks: false,
-        })
-        contentOpts.plugins.push({
-            name: 'enso-on-rebuild',
-            setup: build => {
-                build.onEnd(() => {
-                    console.log('Content bundle updated.')
-                })
-            },
-        })
-        contentOpts.pure.splice(contentOpts.pure.indexOf('assert'), 1)
-        contentOpts.outdir = path.resolve(IDE_DIR_PATH, 'assets')
-        contentOpts.define.REDIRECT_OVERRIDE = JSON.stringify('http://localhost:8080')
-        const contentBuilder = await esbuild.context(contentOpts)
-        const content = await contentBuilder.rebuild()
-        console.log('Result of content bundling: ', content)
-        void contentBuilder.watch()
```

Comment:
How come we don't need this anymore? Do we not need to rebuild the content and dashboard bundles?

Reply:
correct:
- `content/` was the entrypoint for GUI1
- `dashboard/` (as well as GUI2) are now served by GUI2's dev server (the Vite server) which rebuilds with HMR when possible